components/org.apache.stratos.python.cli/src/main/python/cli/CLI.py

```diff
@@ -0,0 +1,190 @@
+from cmd2 import *
+from Utils import *
+from Stratos import *
+import Configs
+from cli.exceptions import AuthenticationError
+
+
+class CLI(Cmd):
+    """Apache Stratos CLI"""
+
+    prompt = Configs.stratos_prompt
+    # resolving the '-' issue
+    Cmd.legalChars = '-' + Cmd.legalChars
+
+    def __init__(self):
+        # resolving the '-' issue
+        [Cmd.shortcuts.update({a[3:].replace('_', '-'): a[3:]}) for a in self.get_names() if a.startswith('do_')]
```

Comment:
Code readability is lesser here. IMO we should go for a cleaner approach with self describing code.